x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/search_fields/search_result.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import VirtualList from 'react-tiny-virtual-list';
+import { EuiButtonEmpty, EuiEmptyPrompt } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { SearchResult as SearchResultType } from '../../../types';
+import { useDispatch } from '../../../mappings_state';
+import { State } from '../../../reducer';
+import { SearchResultItem } from './search_result_item';
+
+interface Props {
+  result: SearchResultType[];
+  documentFieldsState: State['documentFields'];
+}
+
+const ITEM_HEIGHT = 64;
+
+export const SearchResult = React.memo(
+  ({ result, documentFieldsState: { status, fieldToEdit } }: Props) => {
+    const dispatch = useDispatch();
+    const listHeight = Math.min(result.length * ITEM_HEIGHT, 600);
+
+    const clearSearch = () => {
+      dispatch({ type: 'search:update', value: '' });
+    };
+
+    return result.length === 0 ? (
+      <EuiEmptyPrompt
+        iconType="search"
+        title={
+          <h1>
```

Comment:
I think this should be an `h3` since it falls beneath the "Mappings (optional)" title in terms of hierarchy, and that's an `h2`.